hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -381,9 +381,6 @@ public final class OzoneConfigKeys {
   public static final String
       DFS_CONTAINER_RATIS_STATEMACHINEDATA_SYNC_RETRIES =
       ScmConfigKeys.DFS_CONTAINER_RATIS_STATEMACHINEDATA_SYNC_RETRIES;
-  public static final int
-      DFS_CONTAINER_RATIS_STATEMACHINEDATA_SYNC_RETRIES_DEFAULT =
-      ScmConfigKeys.DFS_CONTAINER_RATIS_STATEMACHINEDATA_SYNC_RETRIES_DEFAULT;
```

Comment:
`ScmConfigKeys.DFS_CONTAINER_RATIS_STATEMACHINEDATA_SYNC_RETRIES_DEFAULT` is now unused, can be removed.

Also, I think we should remove the default value for `dfs.container.ratis.statemachinedata.sync.retries` in `ozone-default.xml` (set it to `<value/>`), and also update the description to explain that the default value is calculated based on other properties.

https://github.com/apache/ozone/blob/74de0705c916bb877eae44310f43a0d7069cdef9/hadoop-hdds/common/src/main/resources/ozone-default.xml#L100-L105

Reply:
@adoroszlai Thank you for the review. I have removed the unused configuration and updated the ozone-default.xml with the description. Please let me know if the description can be improved further.